stats.sh

```diff
@@ -52,8 +52,13 @@ END {
 	print "----------------------------------------|------------|--------------|-----------|------------|--------|--------------|-----------"
 	for (p in pname) {
 		pr=pname[p]
-	    arry[proto[pr,3]] = sprintf("%-39s | %10d | %6d %s | %9d | %10d | %6.2f | %12d | %7.2f",pr,proto[pr,2],proto[pr,3],proto[pr,4],proto[pr,5],proto[pr,7],
-			proto[pr,2]*proto[pr,3]/1000000000,proto[pr,2]*proto[pr,5]/10000,proto[pr,3]/proto[pr,7])
+		if (proto[pr,7] == 0) {
```

Comment:
What's this change doing?

Reply:
Benc has a pre-allocation method, therefore are the allocations zero, which would cause an error, because the script is dividing through zero, so if the allocations are zero on a method, it just replaces the division through 'Zero-Allocs'